### PR TITLE
FINERACT-881

### PR DESCRIPTION
**Pull request by xurror**
Remove all hard-coded default passwords

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Commit message starts with the issue number from https://issues.apache.org/jira/projects/FINERACT/. Ex: FINERACT-646 Pockets API.

- [ ] Coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions have been followed.

- [ ] API documentation at https://github.com/apache/fineract/blob/develop/api-docs/apiLive.htm has been updated with details of any API changes.

- [ ] Integration tests have been created/updated for verifying the changes made.

- [ ] All Integrations tests are passing with the new commits.

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the list.)

Our guidelines for code reviews is at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide

### CODE REVIEW

**Comment by vorburger** [comment]
Following #764 and #761 this will pass the build (or not at least not fail because of integration tests) if you rebase this now.

File: kubernetes/fineract-server-deployment.yml [hunk 1]
```diff
@@ -59,7 +60,27 @@ spec:
         image: apache/fineract:latest
         env:
         - name: JAVA_OPTS
```
Comment (vorburger) [inline kubernetes/fineract-server-deployment.yml]: @xurror remove the `JAVA_OPTS` completely, see https://issues.apache.org/jira/browse/FINERACT-900 for why

**Review by vorburger — CHANGES_REQUESTED** [review]

**Comment by xurror** [comment]
@vorburger, think this is good to go now

**Review by vorburger — APPROVED** [review]

**Comment by vorburger** [comment]
_"I'm Lovin' this PR!"_ © ℗ ® ™

**Comment by vorburger** [comment]
PS FTR I haven't actually pulled and myself locally re-verified (I did the original), but I'm sure you have tested this yourself on `minikube` while you made these changes, so I'll merge.

**Pull request merged**